hubstorage/client.py

```diff
@@ -14,6 +17,24 @@
 __version__ = pkgutil.get_data('hubstorage', 'VERSION').strip()
 
 
+logger = logging.getLogger('HubstorageClient')
+
+_ERROR_CODES_TO_RETRY = (429, 503, 504)
```

Comment:
For a global retry code list, I think we should add 408 too. 
More details at http://blog.haproxy.com/2014/05/26/haproxy-and-http-errors-408-in-chrome/ 

We have experienced 408's before ( [#1](https://github.com/scrapinghub/python-hubstorage/pull/27#issuecomment-98152350) and [#2](https://github.com/scrapinghub/python-hubstorage/commit/60039aa70e01d840e0c5524b468c8458eb94ed43))